src/output-json.c

```diff
@@ -713,9 +719,19 @@ OutputCtx *OutputJsonInitCtx(ConfNode *conf)
             }
         }
 
+        ConfNode *meta = ConfNodeLookupChild(conf, "metadata");
+        if (meta != NULL) {
```

Comment:
This means we don't get the filename when using an old configuration file. I think, when processing pcaps, it might make sense to have it by default.